database-mongo/feedData.js

```diff
@@ -0,0 +1,51 @@
+const {db} = require('./index.js');
+var mongoose = require('mongoose');
+var data = require('./seed.js')
+
+//db.overviews.drop();
+
+  var overviewSchema = mongoose.Schema({
+    prod_name: String,
+    specs: Array,
+    included: Array,
+    boxContent: Array,
+    descriptionHeader: Array,
+    description: Array,
+    imageHeader: Array,
+    images: Array,
+    shippingDate: String,
+    details: Array,
+    html: String
+  });
+  
```

Comment:
I think this schema and it's model should still be in the model file.  Maybe export them from Model/index.js and import them here instead?

Reply:
sure i can do that!